src/components/Calculator.js

```diff
@@ -1,49 +1,33 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import calculate from '../logic/calculate';
+import Display from './display';
+import Keypad from './keypad';
 
-const Button = ({ name }) => <td>{name}</td>;
+export default class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: '0',
+      next: null,
+    };
```

Comment:
Just as required in this milestone, we are to make use of `useState` hook to manage our state, `props` to pass data around components, and just as I suggested in the previous comments, you need to convert your class components to functional components to make use of the `useState` hook😉.

<img width="780" alt="image" src="https://github.com/MenendezON/math-magicians/assets/64643995/d3a18db3-62b3-4347-abec-011e180a3b18">